engine/graphics/image.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <vector>
+
+namespace pong
+{
+
+namespace image
+{
+
+std::vector<unsigned char> ConvertAlphaImageToRGBA(const std::vector<unsigned char>& imageData);
+std::vector<unsigned char> FlipImageVertically(const std::vector<unsigned char>& imageData, int width, int height, int comp);
```

Comment:
is it worth it to have an Image class? Seems like a lot of these params are internal hidden details of what an image is. By not encapsulating it, you make the code harder to change later.

Reply:
Yeah that is a good change!